docs/src/main/sphinx/release/release-426.md

```diff
@@ -0,0 +1,52 @@
+# Release 426 (5 Sep 2023)
+
+## General
+
+* Add support for `SET SESSION AUTHORIZATION` and `RESET SESSION AUTHORIZATION`
+  syntax. ({issue}`16067`)
+* Add support for automatic type coercion when creating tables. ({issue}`13994`)
+* Improve performance of decimal aggregations. ({issue}`18868`)
+* Change recommended Java runtime distribution to Eclipse Temurin. ({issue}`18886`)
```

Comment:
This doesn't need a release note item. We don't record changes to documentation in the release notes.